thirdeye-persistence/src/main/java/ai/startree/thirdeye/datalayer/bao/AlertManagerImpl.java

```diff
@@ -38,6 +38,16 @@ public Long loadValue() {
         return countActive();
       }
     });
+    metricRegistry.register("activeTimeseriesMonitoredCount", new CachedGauge<Integer>(15, TimeUnit.MINUTES) {
```

Comment:
In a later PR: Can you centralize all of these cache times into `Constants`. Tomorrow if we want to change 15m to 5m, there shouldn't be a need to go find every metric and change them.

Reply:
Agree on this. Also I was thinking on pulling all these self computing metrics into a central place so that its easy to maintain and gives better visibility. Will create a follow up item on this.